web/src/Installer.test.jsx

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+
+import { act, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { authRender } from "./test-utils";
+import { createClient } from "./lib/client";
+import { PROBING, PROBED, INSTALLING, INSTALLED } from "./lib/client/statuses";
```

Comment:
BTW, it's not clear to me if "status" is a countable or uncountable noun.

Uncountable according to

* https://www.collinsdictionary.com/dictionary/english/status, 5th entry
* https://www.macmillandictionary.com/dictionary/american/status, 3rd entry
* https://dictionary.cambridge.org/dictionary/english/status#cbed-1

Countable according to 

*  https://www.wordreference.com/definition/statuses

Both according to 

* https://english.stackexchange.com/questions/877/what-is-the-plural-form-of-status

:man_shrugging:

Reply:
I highly recommend "status values".

Reply:
Or just "status", as it is the name of a module.